src/engine/AliveSignal.class.js

```diff
@@ -60,7 +60,7 @@ class AliveSignal {
     try {
       const data = JSON.stringify(status)
       const headers = { 'Content-Type': 'application/json' }
-      await this.engine.sendRequest(this.host, 'POST', this.authentication, this.proxy, data, headers)
+      await this.engine.requestService.send(this.host, 'POST', this.authentication, this.proxy, data, headers)
```

Comment:
maybe we should have a method proposed for every North handlers so it could be called this.send or this.HttpSend. Also, to make it more readable and also propose defaults, I propose to pass an object
{ url, mode = Post, authentication etc... }

Reply:
I wanted a minimal impact on other components, but I was already thinking to make some 'alias' methods, like `postJsonValue`, `postFile`, etc. where the parameters are only specific for the caller.
Since AliveSignal also uses the RequestService, I would implement these methods in BaseRequest, but I also can create sendValues() and sendFile() in ProtocolHandler as an additional abstraction level for North developers.
What do you think?